_includes/breadcrumbs.html

```diff
@@ -0,0 +1,23 @@
+<section class="solid-breadcrumbs col-xs-12">
+    {% assign crumbs = page.url | remove:'/index.html' | split: '/' %}
+    <div class="solid-breadcrumbs-container row">
```

Comment:
Perhaps using `<nav>` to indiciate navigation on page? Could also add `aria-label="breadcrumb"` to make it semantically richer.

Reply:
Resolving because it was raised as an issue. (Will also do that for the other comments, so I won't repeat this comment for each of them :) )